launchcontainer/static/html/launchcontainer.html

```diff
@@ -8,5 +9,10 @@
   </form>
   <div id="launcher_notification" class="hide"></div>
   {% if enable_container_resetting %}<button type="button" class="ui-priority-primary" id="launcher_reset">Reset lab</button>{% endif %}
-
+{% else %}
+  {# This message is intended to appear Studio for course authors. #}
+  <p><strong>Error: Unable to connect to the Appsembler Virtual Labs API endpoint.</strong></p>
+  <p>Please configure your Appsembler Virtual Labs API endpoint before using this XBlock.</p>
```

Comment:
```suggestion
  <p>Please contact  <a href="mailto:support@appsembler.com" target="_blank">Appsembler Support</a> so we can configure your Appsembler Virtual Labs API endpoint.</p>
```

I could be wrong, but I don't think that the customer can configure this on his own, right?  My suggested phrasing above captures that.  If he can, I think we should talk with Matthew about writing some customer docs on that (or maybe the docs already exist).